packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -237,12 +256,23 @@ const helperBehavior = {
       compoundedInterest: state.compoundedInterest,
       interestRate,
       latestInterestUpdate: state.latestInterestUpdate,
-      totalDebt: state.totalDebt,
+      // XXX move to governance and type as present with null
       liquidatorInstance: state.liquidatorInstance,
     });
     state.assetUpdater.updateState(payload);
   },
 
+  /** @param {MethodContext} context */
+  metricsNotify: ({ state }) => {
```

Comment:
I think `metricNotify` should be called on `liquidateAll()`, but I don't see how it will be.

Do individual liquidations invoke it?

Reply:
They should. 41fa373c0f7660122cbce86b5f607c8b526a5009

though I ran into an accounting issue that isn't obvious to solve and will be changed by https://github.com/Agoric/agoric-sdk/pull/5343 so I'll come back to this after that.